### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by irinastoica19**

### CODE REVIEW

File: choices.c [hunk 1]
```diff
@@ -0,0 +1,36 @@
+//
+// Created by Irina on 07-Nov-19.
+//
+
+#include <stdio.h>
+#define NR_FOODS 3
+#define NR_DRINKS 5
```
Comment (Ranapop) [inline choices.c]: why are these defined here?

File: display_order.c [hunk 2]
```diff
@@ -0,0 +1,77 @@
+//
+// Created by Irina on 07-Nov-19.
+//
+
+#include <stdio.h>
+#define NR_DRINKS 5
+#define NR_FOODS 3
```
Comment (Ranapop) [inline display_order.c]: why are these defined here? if you don't want them to define them in main, just create a constants.h and define all your constants there...

File: display_order.c [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+    printf("Do you want cutlery?\n");
+    printf("a) Yes\nb) No, thanks!\nc) Go back\n");
+}
+void displayConfirmingOrder(int prices[3][4], int foodChoice, int foodTypeChoice, int pricesDrinks[5], int drinkChoice){
+    printf("Payment amount: %d\n", prices[foodChoice][foodTypeChoice] + pricesDrinks[drinkChoice]);
+    printf("-------------\n");
+    printf("a) Confirm order\n");
+    printf("b) Go back\n");
+}
+void displayOrder(char username [20],char food[][10], int foodChoice, char foodTypes[3][4][20], int foodTypeChoice, int prices[3][4], int drinkChoice, char drinks[][20], int pricesDrinks[5]){
```
Comment (Ranapop) [inline display_order.c]: you're sending too much data

File: display_order.c [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+    printf("-------------\n");
+    printf("Name: %s\n", username);
+    printf("Food items:\n");
+    printf("--- %s %s (%d)\n", food[foodChoice], foodTypes[foodChoice][foodTypeChoice],prices[foodChoice][foodTypeChoice]);
+}
+void displayCutlery(){
+    printf("Do you want cutlery?\n");
+    printf("a) Yes\nb) No, thanks!\nc) Go back\n");
+}
+void displayConfirmingOrder(int prices[3][4], int foodChoice, int foodTypeChoice, int pricesDrinks[5], int drinkChoice){
```
Comment (Ranapop) [inline display_order.c]: sending too much data

File: display_order.c [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+    printf("Payment amount: %d\n", prices[foodChoice][foodTypeChoice] + pricesDrinks[drinkChoice]);
+    printf("-------------\n");
```
Comment (Ranapop) [inline display_order.c]: this belongs to a different function

File: display_order.c [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+        printf("--- %s (%d)\n", drinks[drinkChoice], pricesDrinks[drinkChoice]);
+    }
+}
+void displayCutleryOrder(int cutleryChoice, char addInfo[200]){
+    printf("Cutlery:");
+    if (cutleryChoice == 1)
+    printf("yes\n");
+    else
+    printf("no\n");
+    if (addInfo[0] != NULL) {
```
Comment (Ranapop) [inline display_order.c]: formatting

File: main.c [hunk 7]
```diff
@@ -1,164 +1,63 @@
...
-            {21, 23, 19},
-            {23, 21},
-            {23, 22, 19, 21}
-    };
-
-    int noDrinks= 5;
+    char foodTypes[3][4][20] = { {"Carbonara", "Diavola", "Margherita"}, {"Chicken alfredo", "Mac&cheese"}, {"Tuna", "Chicken", "Greek", "Cobb"} };
     char drinks[][20] = {"Coca Cola", "Fanta", "Lipton", "Water", "No, thanks!"};
+    char addInfo[200], username[20], password[20];
+    int prices[3][4] = {{21, 23, 19}, {23, 21}, {23, 22, 19, 21} };
```
Comment (Ranapop) [inline main.c]: formatting

File: main.c [hunk 8]
```diff
@@ -1,164 +1,63 @@
...
-                if(addInfo[0]!=NULL){
-                    printf("Additional info: ");
-                    puts(addInfo);
-                }
-                printf("Payment amount: %d\n", prices[foodChoice][foodTypeChoice] + pricesDrinks[drinkChoice]);
-                printf("-------------\n");
-                printf("a) Confirm order\n");
-                printf("b) Go back\n");
+                displayOrder(username, food,foodChoice,foodTypes,foodTypeChoice,prices,drinkChoice, drinks, pricesDrinks);
+                displayCutleryOrder(cutleryChoice, addInfo);
```
Comment (Ranapop) [inline main.c]: should be called from displayOrder

**Review by Ranapop — CHANGES_REQUESTED** [review]
See code comments

File: display_order.c (hunk 2 above)
Comment (irinastoica19) [reply display_order.c]: I created a constants.h file. I hope it is OK now.

File: display_order.c (hunk 3 above)
Comment (irinastoica19) [reply display_order.c]: I see. I don't know how I can make it better tho.

**Pull request merged**